src/main/java/com/atlassian/shele/shelePlugin/ao/IssueEntity.java

```diff
@@ -19,11 +18,9 @@
 
 
     Integer getIssueId();
-
     void  setIssueId(Integer issueId);
 
-    @Default("default")
-    Issue getField();
+    String getField();
     void setField(String field);
 
```

Comment:
remove empty lines. Only one line between methods is allowed

Reply:
fixed